src/app/containers/Copyright/index.test.jsx

```diff
@@ -13,6 +13,7 @@ const newsServiceContextStub = {
   imageCopyrightOffscreenText: 'Image source, ',
 };
 const persianServiceContextStub = {
+  lang: 'en-GB',
```

Comment:
I believe this should have a lang of farsi
```suggestion
  lang: 'fa',
```